King-Benx/news-watchtower-5
# Initial application structure [Delivers #163844359]

https://www.pivotaltracker.com/story/show/163844359

**Overview**
This is the initial structure of the android application.

**Background**
NONE

**Acceptance Criteria**
- Default android application for new application

**How to manually test**
- Clone the repository
- Checkout to the ch-initial-app-structure-163844359
- Run ./gradlew build
- Run the application on an emulator or device
- The hello world should be shown when the application loads.

**Story**
[#163844359]
